### PR TITLE
[Delivers #108795040] Move NCR Organizations to database

### PR DESCRIPTION
**Pull request by jessieay**
* Were being read from CSV and loaded into memory
* This is a refactor, no behavioral change
* Story: https://www.pivotaltracker.com/story/show/108795040

### CODE REVIEW

File: spec/db/chores/roles_conversion_spec.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require_relative "../../../db/chores/roles_conversion"
```
Comment (pkarman) [inline spec/db/chores/roles_conversion_spec.rb]: this seems brittle. what happens when we (inevitably) reorganize the file structure of the tests?

File: spec/db/chores/ncr_organization_importer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require_relative "../../../db/chores/ncr_organization_importer"
```
Comment (pkarman) [inline spec/db/chores/ncr_organization_importer_spec.rb]: see previous comment about brittle relative path names.

File: spec/db/chores/ncr_work_order_organization_reassigner_spec.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
+require_relative "../../../db/chores/ncr_work_order_organization_reassigner"
```
Comment (pkarman) [inline spec/db/chores/ncr_work_order_organization_reassigner_spec.rb]: see previous comment about brittle relative path names.

**Comment by pkarman** [comment]
@jessieay looks great. Just a comment about using relative paths in test files -- maybe use `Rails.root` instead?

**Comment by jessieay** [comment]
@pkarman good idea! Updated relative paths at 02ca38a3329c2b9ad0791291f86e71d2db35eec1

**Comment by jessieay** [comment]
Something to look for when QAing: I noticed a few times that when I ran the migration it removed all of the approver names from my proposal. I am not sure why this would happen and it seems to correct itself, but there is something weird there.

**Comment by pkarman** [comment]
great I will QA

**Comment by pkarman** [comment]
I QA'd on c2-dev.

I found a weird bug similar to the one you described. If I change the NCR Organization by modifying the proposal after I have created it, then it re-orders the Approval Steps and drops one.
screen shot 2015-12-09 at 10 48 16 am
screen shot 2015-12-09 at 10 51 26 am
screen shot 2015-12-09 at 10 49 10 am
screen shot 2015-12-09 at 10 49 21 am

**Comment by jessieay** [comment]
Closing this in favor of https://github.com/18F/C2/pull/870

870 is similar but a lighter touch - still adds NCR orgs to the database but keeps `org_code` as a string on `work_order`. This is to prevent the migration from doing the weird thing where it removes approvers from work orders. We should eventually move to put a foreign key on work orders rather than storing a string, but this is a good first step

**Comment by jessieay** [comment]
pushed to c2-dev and rolled back, so dev is no longer broken :)

**Comment by jessieay** [comment]
oh wait, it is broken. somehow the migration rollback via ssh didn't work. re-pushed master to c2-dev and it is broken again. HALP!

**Pull request closed without merging**